benchmarks/benchmark_sd.py

```diff
@@ -0,0 +1,73 @@
+import argparse
+import os
+import sys
+
+import torch
+
+from diffusers import DiffusionPipeline
+
+
+sys.path.append(".")
+from benchmark_utils import (  # noqa: E402
+    BASE_PATH,
+    PROMPT,
+    BenchmarkInfo,
+    benchmark_fn,
+    bytes_to_giga_bytes,
+    generate_csv_dict,
+    write_to_csv,
+)
+
+
+CKPT = "CompVis/stable-diffusion-v1-4"
+
+
+def load_pipeline(run_compile=False):
+    pipe = DiffusionPipeline.from_pretrained(CKPT, torch_dtype=torch.float16, use_safetensors=True)
```

Comment:
Will CKPT be turned into an argument to the script?

Reply:
I was thinking to keep it a constant and separately write out the benchmarking script following what I described here: https://github.com/huggingface/diffusers/pull/5839#discussion_r1407054903.